engine/src/main/resources/assets/i18n/menu_en.lang

```diff
@@ -32,6 +32,7 @@
     "binding-nui-editor": "NUI Editor",
     "binding-nui-skin-editor": "NUI Skin Editor",
     "binding-pause": "In-game Menu",
+    "binding-screenshot": "Capture Screenshot",
```

Comment:
Ah, I didn't realize it was alphabetical. Maybe add a comment on top to indicate the same?